modules/solid_mechanics/doc/content/source/materials/CappedWeakPlaneStressUpdate.md

```diff
@@ -1 +1,791 @@
-!template load file=stubs/moose_object.md.template name=CappedWeakPlaneStressUpdate syntax=/Materials/CappedWeakPlaneStressUpdate
\ No newline at end of file
+# CappedWeakPlaneStressUpdate
+
+!syntax description /Materials/CappedWeakPlaneStressUpdate
+
+## Theory id=theory
+
+Weak-plane plasticity is designed to simulate a layered material.
+Each layer can slip over adjacent layers, and be separated from those
+adjacent layers.  An example of particular interest is
+stratified rocks, which consist of large sheets of fairly strong rock
+separated by weak and very thin joints.  Upon application of stress,
+the joints can fail, either by slipping or separating.  The idea is to
+use one finite element that potentially contains many layers, and
+prescribe ``weak plane plasticity'' for that finite element, so that
+it can fail by joint separation and joint slipping.
+
+Denote the normal to the layers by $z$, and the tangential directions
+by $x$ and $y$.  It is convenient to introduce two new stress variables in
+terms of the stress tensor $\sigma$:
+
+\begin{equation}
+p = \sigma_{zz} \ \ \ \text{and}\ \ \ q = \sqrt{\sigma_{xz}^{2} +
+  \sigma_{yz}^{2}} \ .
+\label{eqn.defn.p.q}
+\end{equation}
+
+In standard elasticity, the stress tensor is symmetric, so an
+equivalent definition of $q$ is
+$q=\sqrt{\text{$\frac{1}{2}$}(\sigma_{xz}+\sigma_{zx})^{2} +
+  \text{$\frac{1}{2}$}(\sigma_{yz}+\sigma_{zy})^{2}}$, however the
+symmetrization is deliberately not written in
+Eqn~[eqn.defn.p.q] and below so that the equations also hold for
```

Comment:
These are now being captured in the text. However, the `Eqn~` is not required as it duplicates text:
<img width="91" alt="Screenshot 2024-08-22 at 10 07 30 AM" src="https://github.com/user-attachments/assets/0a04d190-0428-4839-a893-1ece1a831e01">

Reply:
changed
ugh even the ~ does not render right